raster_cutter.py

```diff
@@ -430,37 +499,45 @@ def delete_world_file(directory_url):
     if os.path.exists(worldfile_path):
         os.remove(worldfile_path)
     else:
-        raise Exception("The file does not exist")
+        raise Exception('The file does not exist')
 
 
 def get_worldfile_url_from_dir(directory_url):
-    index = directory_url.find(".")
+    index = directory_url.find('.')
     if index != -1:
         worldfile_path = directory_url[:index]
-        worldfile_path += ".wld"
+        worldfile_path += '.wld'
     else:
-        raise Exception("Could not find . in path")
+        raise Exception('Could not find . in path')
     return worldfile_path
 
+
 def pre_launch_checks():
```

Comment:
This function doesn't seem to be used anywhere, if not required by magic (pyqt), remov it.